README.md

```
@@ -26,45 +26,68 @@ Before running the project, ensure that you have the following installed:
+5. **Install Air**
    To install Air, follow the installation steps here:
    [Air Installation Guide](https://github.com/air-verse/air)
 
-## Setup env
-1. Create .env file in root directory
-2. Copy .env.example content in .env
+### Discord Bot Setup
+ - Refer to [Setup](./SETUP.md)
 
-## Running RabbitMQ with Docker
+### Setup env
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)